src/commands/stopVirtualMachine.ts

```diff
@@ -16,14 +16,14 @@ export async function stopVirtualMachine(context: IActionContext, node?: Virtual
     }
 
     const computeClient: ComputeManagementClient = createAzureClient(node.root, ComputeManagementClient);
-    const stoppingVm: string = localize('stoppingVm', `Stopping "${node.name}"...`);
-    const stoppedVm: string = localize('stoppedVm', `"${node.name}" has been stopped.`);
+    const deallocatingVm: string = localize('deallocatingVm', `Deallocating "${node.name}"...`);
+    const deallocatedVm: string = localize('deallocatedVm', `"${node.name}" has been deallocated.`);
```

Comment:
nit: I'm not a fan of all these local variables for strings. It means you have to change the name of the variable every time you change the string and the two variables looks so similar I feel like it's easy to get them mixed up. I'd much rather see this below:

```
ext.outputChannel.appendLog(localize('deallocatingVm', `Deallocating "${node.name}"...`));
```